src/config.ts

```diff
@@ -16,12 +16,16 @@ export interface Layer {
 
 export interface Story {
   title: string,
-  content: string
+  content: string,
 }
 
 export interface ControlProperty {
   id: string,
-  type: 'slider',
+  type?: 'slider'|'flyTo',
+  title?: string,
+  subtitle?: string,
   key?: string,
-  prefixKey?: string
+  prefixKey?: string,
+  // generic arguments that depends on the type
+  args: any

```

Comment:
I would really like to avoid using `any`

Reply:
I have a fix in another branch to remove this. We need to add some more pieces to this config, so I started splitting it out to arg types by control type (for example, a slider has slider specific arguments). In an effort to keep this PR small, can we keep this `any` and I will remove it in one of my next PRs?